<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1510870662766" stop="1510870667366" version="1.4.23" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Test runner : Package with subpackages</name>
    <title>Test runner : Package with subpackages</title>
    <test-cases>
        <test-case start="1510870662889" stop="1510870666615" status="failed">
            <name>checkStatusCodeOfRequestWithInvalidCityName</name>
            <description>Request with invalid cityName and valid appid parameters. Response should be 404</description>
            <failure>
                <message>AssertionError: Actual statusCode doesn`t match with expected expected [404] but found [200]</message>
                <stack-trace>java.lang.AssertionError: Actual statusCode doesn`t match with expected expected [404] but found [200]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit.checkStatusCodeOfRequestWithInvalidCityName(WeatherDataForOneLocationByCityNameTestSuit.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments>
                <attachment title="request" source="bdde415a-e51a-42ca-b06d-ede778314b15-attachment.txt" type="text/plain"/>
                <attachment title="response" source="a8d79a60-390e-4555-9fb8-52bec45514de-attachment.txt" type="text/plain"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-14LIAQ3"/>
                <label name="thread" value="8740@DESKTOP-14LIAQ3.TestNG-test=Package with subpackages-1(14)"/>
            </labels>
        </test-case>
        <test-case start="1510870666617" stop="1510870666734" status="failed">
            <name>checkStatusCodeOfRequestWithMissedCityName</name>
            <description>Request with missed cityName but valid appid parameters. Response should be 404</description>
            <failure>
                <message>AssertionError: Actual statusCode doesn`t match with expected expected [401] but found [200]</message>
                <stack-trace>java.lang.AssertionError: Actual statusCode doesn`t match with expected expected [401] but found [200]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit.checkStatusCodeOfRequestWithMissedCityName(WeatherDataForOneLocationByCityNameTestSuit.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments>
                <attachment title="request" source="8608c92b-c365-400b-b67a-b6afe1b304c9-attachment.txt" type="text/plain"/>
                <attachment title="response" source="f2edbabf-cccb-44a7-b166-6187318f21ab-attachment.txt" type="text/plain"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-14LIAQ3"/>
                <label name="thread" value="8740@DESKTOP-14LIAQ3.TestNG-test=Package with subpackages-1(14)"/>
            </labels>
        </test-case>
        <test-case start="1510870666735" stop="1510870666819" status="failed">
            <name>checkStatusCodeOfRequestWithMissedCityNameAndAppId</name>
            <description>Request with missed cityName and appid parameters. Response should be 404</description>
            <failure>
                <message>AssertionError: Actual statusCode doesn`t match with expected expected [401] but found [200]</message>
                <stack-trace>java.lang.AssertionError: Actual statusCode doesn`t match with expected expected [401] but found [200]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit.checkStatusCodeOfRequestWithMissedCityNameAndAppId(WeatherDataForOneLocationByCityNameTestSuit.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments>
                <attachment title="request" source="81cad000-cacd-4093-9716-a12be3b52181-attachment.txt" type="text/plain"/>
                <attachment title="response" source="312cef6f-8381-4704-ba39-a31a81767e58-attachment.txt" type="text/plain"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-14LIAQ3"/>
                <label name="thread" value="8740@DESKTOP-14LIAQ3.TestNG-test=Package with subpackages-1(14)"/>
            </labels>
        </test-case>
        <test-case start="1510870666820" stop="1510870666915" status="passed">
            <name>checkStatusCodeOfRequestWithValidData</name>
            <description>Request with valid data in cityName and appid parameters. Response should be 200</description>
            <steps/>
            <attachments>
                <attachment title="request" source="3557e9fa-011b-439b-82d3-cd15263ad443-attachment.txt" type="text/plain"/>
                <attachment title="response" source="d3ccd854-4f00-4ab1-b997-e9302ad3bb8c-attachment.txt" type="text/plain"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-14LIAQ3"/>
                <label name="thread" value="8740@DESKTOP-14LIAQ3.TestNG-test=Package with subpackages-1(14)"/>
            </labels>
        </test-case>
        <test-case start="1510870666919" stop="1510870667318" status="passed">
            <name>checkWeatherIdWithValidData</name>
            <description>Request with valid data in cityName and appid parameters. WeatherID shoud be 2643743</description>
            <steps/>
            <attachments>
                <attachment title="request" source="c0bcbbda-87c8-4e5a-929a-d59471e6b5c7-attachment.txt" type="text/plain"/>
                <attachment title="response" source="90244657-4c3a-47f6-9d36-065a884019fb-attachment.txt" type="text/plain"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-14LIAQ3"/>
                <label name="thread" value="8740@DESKTOP-14LIAQ3.TestNG-test=Package with subpackages-1(14)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
